denizen_scripts/survival/repo-link/global_systems/warps.dsc

```diff
@@ -206,43 +206,43 @@ favorite_warps:
     filler: white_stained_glass_pane[display_name=<&a>]
     back_button: <item[barrier].with[display_name=<&c>Close<&sp>GUI;nbt=action/back]>
   slots:
-   - "[filler] [filler] [filler] [filler] [filler] [filler] [filler] [filler] [filler]"
-   - "[filler] [] [] [] [] [] [] [] [filler]"
-   - "[filler] [] [] [] [] [] [] [] [filler]"
-   - "[filler] [] [] [] [] [] [] [] [filler]"
-   - "[filler] [filler] [filler] [filler] [filler] [filler] [filler] [filler] [filler]"
-   - "[filler] [filler] [filler] [filler] [back_button] [filler] [filler] [filler] [filler]"
+  - "[filler] [filler] [filler] [filler] [filler] [filler] [filler] [filler] [filler]"
+  - "[filler] [] [] [] [] [] [] [] [filler]"
+  - "[filler] [] [] [] [] [] [] [] [filler]"
+  - "[filler] [] [] [] [] [] [] [] [filler]"
+  - "[filler] [filler] [filler] [filler] [filler] [filler] [filler] [filler] [filler]"
+  - "[filler] [filler] [filler] [filler] [back_button] [filler] [filler] [filler] [filler]"
 
 favorite_warps_open:
   type: task
   script:
-    - foreach <yaml[player.<player.uuid>].list_keys[warps.favorites]||<list>>:
-      - define identifier <yaml[player.<player.uuid>].read[warps.favorites.<[value]>]>
-      - define type personal
-      - inject build_warp_item
-      - define "list:|:<[item].with[display_name=<&e><[value]>;nbt=name/<[value]>;lore=<[item].lore.remove[first].include[<&e>----------------|<&a>Left Click to Warp.|<&c>Right Click to Remove]>]>"
-    - define inventory <inventory[favorite_warps]>
-    - if <[list]||null> != null:
-      - give <[list]> to:<[inventory]>
-    - inventory open d:<[inventory]>
+  - foreach <yaml[player.<player.uuid>].list_keys[warps.favorites]||<list>>:
+    - define identifier <yaml[player.<player.uuid>].read[warps.favorites.<[value]>]>
+    - define type personal
+    - inject build_warp_item
+    - define "list:|:<[item].with[display_name=<&e><[value]>;nbt=name/<[value]>;lore=<[item].lore.remove[first].include[<&e>----------------|<&a>Left Click to Warp.|<&c>Right Click to Remove]>]>"
+  - define inventory <inventory[favorite_warps]>
+  - if <[list]||null> != null:
```

Comment:
Something we may do here instead of checking if a ListTag isn't equal to `null` based on a fallback is checking whether the definition exists or not, or pre-defining `list` as an empty list before the `foreach` (`- define list <list>`), and checking `if <[list].is_empty>` to avoid over-using fallbacks